internal/serverapi/serverapi.go

```diff
@@ -154,3 +155,24 @@ type Snapshot struct {
 type SnapshotsResponse struct {
 	Snapshots []*Snapshot `json:"snapshots"`
 }
+
+// MountSnapshotRequest contains request to mount a snapshot.
+type MountSnapshotRequest struct {
+	Root string `json:"root"`
+}
+
+// UnmountSnapshotRequest contains request to unmount a snapshot.
+type UnmountSnapshotRequest struct {
+	Root string `json:"root"`
+}
+
+// MountedSnapshot describes single mounted snapshot.
+type MountedSnapshot struct {
+	Path string    `json:"path"`
+	Root object.ID `json:"root"`
+}
+
+// MountedSnapshots describes single mounted snapshot.
+type MountedSnapshots struct {
+	Items []*MountedSnapshot `json:"items"`
```

Comment:
Curious, any reason for making this a pointer and not making this a copy? as in:

```go
	Items []MountedSnapshot `json:"items"`
```